subsys/bluetooth/host/keys.h

```diff
@@ -102,16 +102,25 @@ enum {
 
 struct bt_keys_link_key {
 	bt_addr_t               addr;
+	u8_t                    storage_start[0]  __aligned(sizeof(void *));
 	u8_t                    flags;
+#if (defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST))
+	u32_t                   aging_counter;
```

Comment:
Sorry for not making this comment earlier:
I'm wondering if it would be smartest to put at the end of the storage, in case we want to support enabling this feature, doing a firmware update and work with existing bonds.